src/plots/gl3d/scene.js

```diff
@@ -383,6 +383,13 @@ function computeTraceBounds(scene, trace, bounds) {
     }
 }
 
+function isCloseToZero(a) {
+    if(Math.abs(a) > Number.MIN_VALUE) { // the smallest positive numeric value representable in JavaScript
```

Comment:
I suppose that's a looser condition than `if(a !== 0)` but the numbers you're subtracting need to already be extremely small in order for their difference to be exactly `Number.MIN_VALUE`. I suspect we actually want to be substantially looser than that.

Exactly how that should happen I'm not sure, but I think the criterion should be something that depends on the magnitude of the numbers, to the effect of "there should be at least N representable numbers between a and b" where N is maybe 100.

see eg #179 and #1070

Reply:
@archmoj did adding this `isCloseToZero` fix a particular bug? If so, could you provide the problematic `"data"` and `"layout"` ?

Reply:
> This way it would be easier to investigate if other modifications may have any side effect.

Great thinking. Thanks!